src/receivers/MCServer.hs

```diff
@@ -81,9 +81,10 @@ fetchServerDetails server_ip = do
                 let playersmax = show $ players_max $ fromJust $ players serverDeets
                 let motdclean = alwaysHead $ clean $ fromJust $ motd serverDeets
                 let ipstr = ip serverDeets
+                let ver = version serverDeets
                 let ann2 = "Current Players: " <> playersonline <> "/" <> playersmax <> ".\n"
                 let ann3 = "Message of the Day: *" <> motdclean <> "*\n"
-                let ann4 = "Come join at `" <> ipstr <> "`"
+                let ann4 = "Come join at `" <> ipstr <> "` on version `" <> ver <> "`"
```

Comment:
`ver` would be a Maybe String here, being Nothing if the server is offline or non-existent.